cmd/antctl/main.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2019 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"flag"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+	"k8s.io/component-base/logs"
+
+	"github.com/vmware-tanzu/antrea/pkg/antctl"
+)
+
+var (
+	commandName = path.Base(os.Args[0])
+	inPod       = len(os.Getenv("POD_NAME")) != 0
+	isAgent     = strings.HasPrefix(os.Getenv("POD_NAME"), "antrea-agent")
+)
+
+var rootCmd = &cobra.Command{
+	Use:   commandName,
+	Short: commandName + " is the command line tool for Antrea",
+	Long:  commandName + " is the command line tool for Antrea that supports showing runtime information of ${component}",
```

Comment:
Maybe changed to a generic message, like status query and configuration setting, as we might introduce read-write cmd.

Reply:
Changed to`antctl is the command line tool for Antrea that supports querying status and setting configuration of ${component}`. How about this?